apiservers/engine/API-COMPATIBILITY.md

```diff
@@ -1,7 +1,73 @@
 # Docker Remote API Compatibility
 VIC attempts to be compatible with the Docker remote API; however, there are some differences.  The API differences are documented below, grouped together by functional areas.
 
-## Container APIs
+# support Matrix
+
+## Container API Support
+
+|API|support?|
+|---|---|
+|List Containers|YES|
+|Create a Container|YES|
+|inspect a container| YES|
+|List processes running inside a container|YES|
+|Get Container Logs|YES|
+|Inspect changes on a containers filesystem|YES|
```

Comment:
I'm not sure inspecting changes on the container filesystem is a priority.  The user scenarios for this functionality is so limited.  We'll eventually do it, but I'm not sure it'll show up in VIC 1.0.